Queryable/Queryable/PhotoHelper/PhotoLibrary.swift

```diff
@@ -7,7 +7,7 @@ import os.log
 
 class PhotoLibrary {
     static func checkAuthorization(status: PHAuthorizationStatus = PHPhotoLibrary.authorizationStatus(for: .readWrite)) async -> Bool {
-        switch PHPhotoLibrary.authorizationStatus(for: .readWrite) {
+        switch status {
```

Comment:
whoops almost forgot to swap this out

I think I don't like using the default argument in this way, but waiting for feedback re: logs & style before changing anything